src/app/_components/Table/TableHeaderWithTooltip.tsx

```diff
@@ -0,0 +1,54 @@
+import { Button } from '@headlessui/react'
+import { Question } from '@phosphor-icons/react/dist/ssr'
+import { clsx } from 'clsx'
+
+import { ClickTooltip, type ClickTooltipProps } from '@/components/ClickTooltip'
+import { Icon } from '@/components/Icon'
+
+type TableHeaderWithTooltipProps = {
+  header: string
+  description: ClickTooltipProps['description']
+}
+
+const touchTarget = {
+  iconSize: 20,
+  iconWrapperPadding: 'p-1.5',
+  class: 'p-2',
+  offsetClass: '-m-2',
+}
+
+export function TableHeaderWithTooltip({
+  header,
+  description,
+}: TableHeaderWithTooltipProps) {
+  return (
+    <div className="flex items-center gap-1">
+      <span>{header}</span>
+      <ClickTooltip description={description} gap={0}>
```

Comment:
What's the `gap` prop here about? And why does the parent need control over it?

Reply:
Also, we have `gap = 8` as the fallback but this is the only time we use it? Do we need it as a prop?

Reply:
We keep the gap at 0 for now

Reply:
✅